src/Platforms/SQLServer2012Platform.php

```diff
@@ -5,6 +5,8 @@
 /**
  * Provides the behavior, features and SQL dialect of the Microsoft SQL Server database platform
  * of the oldest supported version.
+ *
+ * @deprecated Use {@see SQLServerPlatform} instead.
```

Comment:
Not sure if `{@see ...}` also works but we usually use `{@link ...}` to reference code elements.